src/components/GamesList/GamesList.css

```diff
@@ -0,0 +1,7 @@
+.ui.black.inverted.right.aligned.segment {
+    transform: none;
+}
```

Comment:
There's and interesting underlying issue here. I am using the class ".right" somewhere else and it's creating this bug. The solution should be to rename the class I am using elsewhere to get rid of this conflict.

Long term, the solution might be to start using something like `styled-components` to avoid these kinds of issues in future.

Reply:
I still think this should be addressed. I understand what you're doing here but it's going to be confusing later and, until that other class named `right` gets fixed we are going to just keep having this issue.

Reply:
So should I just eliminate this and wait for you to fix the component? Or is there a better solution?